Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -240,10 +241,16 @@ getDriverMobileNumber driverNumberType callSid callFrom_ callTo_ _dtmfNumber cal
     findRiderDetails merchantId mobileNumberHash =
       runInReplica (Person.findByRoleAndMobileNumberAndMerchantId USER "+91" mobileNumberHash merchantId) >>= \case
         Nothing -> pure Nothing
-        Just person ->
-          QRB.findAssignedByRiderId person.id >>= \case
+        Just person -> do
+          selfActiveBooking <- QB.findAssignedByRiderId person.id
+          partyActiveBooking <-
```

Comment:
We can avoid this query if selfActiveBooking is found in most of the OneWay cases usually

Reply:
✅